src/crewai/crew.py

```diff
@@ -154,6 +155,10 @@ class Crew(BaseModel):
         default=False,
         description="Plan the crew execution and add the plan to the crew.",
     )
+    planning_llm: Optional[Any] = Field(
```

Comment:
I can't get pytest or crews to run with this new property.

Please check out the loom video for more info:

https://www.loom.com/share/2aece814e603487c9c7dc09c8106f141?sid=4c5ad3ba-94d4-4db9-9377-52dfc369bdff